tests/test_experiment_server.py

```diff
@@ -733,29 +735,33 @@ def test_with_invalid_participant_id_returns_error(self, webapp):
         assert resp.status_code == 403
         assert '/node POST no participant found' in resp.data
 
-    def test_with_valid_participant_creates_participant_node(self, a, webapp):
+    def test_with_valid_participant_creates_participant_node(self, db_session, a, webapp):
         participant_id = a.participant().id
+        db_session.commit()
```

Comment:
I'm guessing the change to db session management necessitated these `commit()`s?

Reply:
Yeah -- the `serialized` decorator now resets the session as it starts, so we now need to be explicit about commiting things that happen in test setup prior to calling a view that uses that decorator. The change to the decorator was motivated by some errors that were happening under high concurrency where it claimed that the session had already been committed. I don't 100% understand what that was about and it's possible that the change I made to when gevent monkey patches get loaded makes it unnecessary. I suppose I can check that pretty easily and that would be worth doing.

Reply:
Ah -- we do need to reset the session, because the command that marks the transaction as using the SERIALIZABLE transaction level needs to be the first command in the transaction.